src/main/java/frc/team2412/robot/commands/arm/SetArmCommand.java

```diff
@@ -31,6 +31,6 @@ public void end(boolean interrupted) {
 
 	@Override
 	public boolean isFinished() {
-		return armSubsystem.isArmNearGoal();
+		return armSubsystem.isArmNearGoal() || armSubsystem.getManualOverride();
```

Comment:
Not that I know of, but good callout that this command is missing `addRequirements` in the constructor. With that missing, that means this command & any other command could both try to use the armSubsystem at the same time, which is generally bad (two commands could try to set a goal at the same time, with the result being apparently random behavior).

Reply:
Note that to prevent this from interrupting manual override (since they both require arm subsystem), we'd need to set the manual override command's interruption behavior to `kCancelIncoming` (which would prevent this and any other arm commands from being scheduled while the manual override is on).